### PR TITLE
Minimum number to form the sum even

### PR DESCRIPTION
**Pull request by anshmishra010**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Given an array and size , the task is to add the minimum number(should be greater than 0) to the array so that the sum of the array becomes even

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#5694

### CODE REVIEW

**Review by bhav09 — CHANGES_REQUESTED** [review]
You need to add comments to your code. Make sure the comments you add are above the line of code and not alongside it.

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]
Resolve merge conflicts as well

**Comment by anshmishra010** [comment]
Mentors as you have requested to add comments , so I have tried my best to add some comments . Still if you like me to add more comment so please mark the line where to add and what to add . PLEASE !

File: C-Plus-Plus/cp/Minimum_number_to_form_the_sum_even.cpp [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    Solution obj;
+    cout << "Minimum Number to be added in the array : " << endl;
+    cout << obj.minNum(a, n);
+
+    return 0;
+}
+
+/*
+Time complexity : O(n)
+Space complexity : On)
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/Minimum_number_to_form_the_sum_even.cpp]: Fix typo in SC

File: C-Plus-Plus/cp/Minimum_number_to_form_the_sum_even.cpp [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+Test Case 2 :
+    Input :
+    Enter the size of the array : 
+    9
+    Enter 9 number of elements : 
+    1 2 3 4 5 6 7 8 9
+
+    Output :
+    Minimum Number to be added in the array : 
+    1
```
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/Minimum_number_to_form_the_sum_even.cpp]: Remove the extra test case one is enough

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

File: C-Plus-Plus/cp/Minimum_number_to_form_the_sum_even.cpp (hunk 2 above)
Comment (anshmishra010) [reply C-Plus-Plus/cp/Minimum_number_to_form_the_sum_even.cpp]: I will remove it but some of the mentors say to add minimum 2 test cases .

**Review by ankitaggarwal23 — APPROVED** [review]

**Pull request merged**